torch/csrc/api/include/torch/nn/modules/any.h

```diff
@@ -552,5 +561,65 @@ ModuleType& AnyModule::get_(
       c10::demangle(typeid(ModuleType).name()));
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ NamedAnyModule ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+class NamedAnyModule {
+ public:
+  /// We don't allow constructing an empty `NamedAnyModule`.
+  NamedAnyModule() = delete;
+
+  /// Creates a `NamedAnyModule` from a (boxed) `Module`.
+  template <typename ModuleType>
+  NamedAnyModule(std::string name, std::shared_ptr<ModuleType> module_ptr) : NamedAnyModule(std::move(name), AnyModule(std::move(module_ptr))) {}
```

Comment:
can you format this nicer?  Look at the entire file -- it's quite obvious what part you wrote :).

Reply:
Fixed